src/main/java/com/switchfully/parkshark_2024_10/division/Division.java

```diff
@@ -1,14 +1,48 @@
 package com.switchfully.parkshark_2024_10.division;
 
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.Id;
+import com.switchfully.parkshark_2024_10.user.Director;
+import jakarta.persistence.*;
 
 @Entity
 public class Division {
     @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE , generator = "division_seq")
+    @SequenceGenerator(name = "division_seq", sequenceName = "division_seq", allocationSize = 1)
     private Long id;
 
+    @Column(name = "name")
+    private String name;
+
+    @OneToOne
+    @JoinColumn(name = "id")
+    private Director director;
+
+    @Column(name = "original_company_name")
+    private String originalCompanyName;
+
+    public Division(String name, Director director, String originalCompanyName) {
+        this.name = name;
+        this.director = director;
+        this.originalCompanyName = originalCompanyName;
+    }
+
+    public Division() {
+
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Director getDirector() {
+        return director;
+    }
+
+    public String getOriginalCompanyName() {
+        return originalCompanyName;
+    }
+
     public void setId(Long id) {
         this.id = id;
     }
```

Comment:
Is this a thing? Not sure, because being able to overwrite the `id` of an existing Entity seems like a bad idea...